s4e-backend/src/main/java/pl/cyfronet/s4e/data/repository/query/QueryBuilder.java

```diff
@@ -0,0 +1,8 @@
+package pl.cyfronet.s4e.data.repository.query;
+
+import java.util.List;
+import java.util.Map;
+
+public interface QueryBuilder {
+    StringBuilder prepareQueryAndParameters(Map<String, Object> params, List<Object> parameters, StringBuilder resultQuery);
```

Comment:
Why return StringBuilder here? You're passing it by reference anyway and discard the returned value.